screens/Navigation/navHelpers.ts

```diff
@@ -56,6 +56,28 @@ export const getConverseStateFromPath =
     if (pathForState?.startsWith("/")) {
       pathForState = pathForState.slice(1);
     }
+
+    // Handle group invite links
+    if (pathForState?.startsWith("group-invite")) {
```

Comment:
I don't think we should need this any more, we should just match whatever the backend is giving us with the app

Reply:
@alexrisch happy to remove this if we're only using Universal Links. Looks like the case at the moment – [see here on the website repo](https://github.com/ephemeraHQ/converse-website/blob/93f1aa5949e1f1231dc4b33086d3fe5bbe2a870a/utils/device.ts#L48-L52) but there is still a commented-out `${scheme}://` for group links specifically – I'm unsure; what do you think?

Is support for the URL app scheme no longer needed?

I did the change below on the website codebase in dev only (not committed) to use app scheme and use the group invite link, as Universal Links were not working locally:
```
: inviteId
-      ? `${url}/group-invite/${inviteId}`
+      ? `${scheme}://group-invite?inviteId=${inviteId}`
```